examples/client/client.c

```diff
@@ -718,8 +718,18 @@ static void ClientRead(WOLFSSL* ssl, char* reply, int replyLen, int mustRead)
 
 static void Usage(void)
 {
-    printf("client "    LIBWOLFSSL_VERSION_STRING
+    printf("wolfSSL client "    LIBWOLFSSL_VERSION_STRING
            " NOTE: All files relative to wolfSSL home dir\n");
+
+    /* print out so that scripts can know what the max supported key size is */
+    printf("Max key size in bits for build is set at : ");
+#ifdef USE_FAST_MATH
+    printf("%d\n", FP_MAX_BITS/2);
+#else
+    /* normal math has unlimited max size */
```

Comment:
What about SP math?

Reply:
Thanks, updated the code to account for SP math only build and for when NO_RSA is defined